antora-ui-camel/src/css/main.css

```diff
@@ -1,5 +1,6 @@
 @media screen and (max-width: 1023px) {
-  aside.toc.sidebar {
+  aside.toc.sidebar,
+  aside.toc-sidebar {
```

Comment:
I need to currently find the issue 'cause whenever I use `<aside class="toc sidebar">` in case of Hugo content, this element doesn't render on the page. That's why I temporarily made use of `<aside class="toc toc-sidebar">`